packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -290,6 +292,11 @@ export const esmPatientRegistrationSchema = {
       },
     },
     dateOfBirth: {
+      allowEstimatedDateOfBirth: {
```

Comment:
I don't understand the difference between `allowEstimatedDateOfBirth` and `useEstimatedDateOfBirth.enabled`. Why do we have both of these?

Reply:
Hi @ibacher !
The useEstimatedDateOfBirth was added by @donaldkibet with the following description

![image](https://github.com/openmrs/openmrs-esm-patient-management/assets/51502471/3ef10c51-9d9e-43d7-9e37-326e4bc1564d)

Reply:
Right, so, maybe to rephrase, if I showed this configuration schema to an implementation team, will they be able to figure this out on their own? One quick fix would be to actually change the description of `useEstimatedDateOfBirth` so that it says "Whether to use a fixed day and month for estimated dates of birth" and related changes to the other two.

The important thing to remember is that these configuration schemas aren't primarily for developers but for implementers, i.e., this is a user interface, and we should make an effort to ensure that our user interfaces are clear and easy-to-use.

Reply:
I agree @ibacher !
Thank you!